components/Popup.tsx

```diff
@@ -0,0 +1,30 @@
+import { Dispatch, FC, SetStateAction, useState } from 'react'
+import { Box } from '@primer/react'
+import { Dialog } from '@primer/react/drafts'
```

Comment:
We have a problem here. You are using a draft.
These are two different Dialogs
```suggestion
import { Dialog } from '@primer/react'
```